src/main/java/uk/ac/ebi/ampt2d/accession/serial/block/persistence/entities/ContiguousIdBlock.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.serial.block.persistence.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.Table;
+
+/**
+ * This class represents a block allocated by a consumer, in a monotonic sequence associated with a category.
+ * <p>
+ * It is defined by the start of the monotonic sequence, the end and the last committed value to the database.
+ * <p>
+ * The last committed value of the block is initialized as the start position of the block minus one, to simplify the
+ * application logic that implements the calculations.
+ */
+@Entity
+@Table(
+        name = "contiguous_id_blocks",
+        indexes = {
+                @Index(name = "CATEGORY_INDEX", columnList = "categoryId")
+        }
+)
+public class ContiguousIdBlock implements Comparable<ContiguousIdBlock> {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false, length = 255)
+    private String categoryId;
+
+    @Column(nullable = false, length = 255)
+    private String applicationInstanceId;
+
+    private long start;
+
+    private long end;
+
+    private long lastCommitted;
+
+    // Create / update dates
```

Comment:
Do we really need these? One thing is to track them for a specific accession, but for an allocated block I find it a bit strange.

Reply:
I think they could help in case of catastrophe, but, as you can see the comment is more in the line of do we add them? As Spring data can handle that part if we want so.